.gitignore

```diff
@@ -2,6 +2,15 @@
 # - A leading slash means the pattern is anchored at the root.
 # - No leading slash means the pattern matches at any depth.
 
+# Sublime files
+*~
+
+# Random files
+get-pip.py
+setup.py-e
+test/play.py
+v/
```

Comment:
Sublime files are fine, I think, like any files with a common pattern unique to an application used by developers.